chapters/statements.tex

```diff
@@ -442,25 +435,21 @@ \subsubsection{Restrictions on Where a When-statement may occur}\label{restricti
 \subsubsection{Restrictions on Statements within When-Statements}\label{restrictions-on-statements-within-when-statements}
 
 \begin{nonnormative}
-In contrast to when-equations, \cref{restrictions-on-equations-within-when-equations},
-there are no additional restrictions within when-statements:
+In contrast to \lstinline!when!-equations, \cref{restrictions-on-equations-within-when-equations}, there are no additional restrictions within \lstinline!when!-statements:
 \begin{itemize}
 \item
   In algorithms, all assignment statements are already restricted to left-hand-side variables.
 \item
-  If at least one element of an array appears on the left-hand-side of the assignment operator inside
-  a when-statement, it is as if the entire array appears in the left-hand-side
-  according to \cref{execution-of-an-algorithm-in-a-model}.
-  Thus, there is no need to restrict the indices to parameter expressions.
+  If at least one element of an array appears on the left-hand-side of the assignment operator inside a \lstinline!when!-statement, it is as if the entire array appears in the left-hand-side according to \cref{execution-of-an-algorithm-in-a-model}.
+  Thus, there is no need to restrict the indices to parameter-expressions.
 \item
-  For-clauses and if-clauses are not problematic inside when-statements in algorithms, since
-  all left-hand-side variables inside when-statements are assigned to their pre-values
-  before the start of the algorithm, according to \cref{execution-of-an-algorithm-in-a-model}.
+  \lstinline!for!-loops and \lstinline!if!-statements are not problematic inside \lstinline!when!-statements in algorithms, since all left-hand-side variables inside \lstinline!when!-statements are assigned to their pre-values before the start of the algorithm, according to \cref{execution-of-an-algorithm-in-a-model}.
```

Comment:
```suggestion
  The \lstinline!for!-loops and \lstinline!if!-statements are not problematic inside \lstinline!when!-statements in algorithms, since all left-hand-side variables inside \lstinline!when!-statements are assigned to their pre-values before the start of the algorithm, according to \cref{execution-of-an-algorithm-in-a-model}.
```